tools/reactive-controllers/element-resolution.md

```diff
@@ -0,0 +1,82 @@
+## Description
+
+An `ElementResolutionController` keeps an active reference to another element in the same DOM tree. Supply the controller with the a selector by which it can locate the element to reference and it will manage observing the DOM tree in question to ensure that the reference it holds is always the first element that matches that selector or `null`.
```

Comment:
```suggestion
An `ElementResolutionController` keeps an active reference to another element in the same DOM tree. Supply the controller with a selector that matches the element and it will manage observing the DOM tree to ensure that the reference it holds is always the first matched element or `null`.
```